lib/index.js

```diff
@@ -68,8 +69,27 @@ module.exports = class DockerComposeParser {
    */
   static _parseDockerComposeFile (dockerComposeFileString) {
     return Promise.try(() => yaml.safeLoad(dockerComposeFileString))
-    .tap(dockerCompose => validate(dockerComposeFileSchema, dockerCompose))
-    .then(dockerCompose => dockerCompose.services)
+      .tap(dockerCompose => validate(dockerComposeFileSchema, dockerCompose))
+      .then(dockerCompose => dockerCompose.services)
+  }
+
+  /**
+   * Adds a main service if one doesn't already exist
+   *
+   * @param {Object<String:Service>} services - Object with service names as key
+   * @param {String}                 repoName - name of the repo
+   * @private
+   */
+  static _checkIfMissingMain (services, repoName) {
+    const mainServiceName = DockerComposeParser._getMainName(services)
+    if (!mainServiceName) {
```

Comment:
Why not just randomly select one as main? Why a `busybox` we won't use anyways...